### DISCUSSION TITLE
Searching inside a particular node and child nodes only

### DISCUSSION THREAD

**Discussion by manasmbellani** [freeplane/freeplane#750] [Help]
Hi, 
In a large freeplane mind map, I want to select a node and search for the presence of the text in that node, and all of its child nodes only. Is it possible to do that in Freeplane?  

By default, When I am selecting a node and trying to find text using Find or Find and Replace options, I am getting search results from the entire mindmap instead of just for occurrences of text within that node or any of its child nodes. 

Thanks.

**Comment by sifran-github** [comment]
one option is to filter the currently selected node, and then search.

![image](https://user-images.githubusercontent.com/5730355/195994592-3351e464-e342-490c-b77b-2d39866e7916.png)

**Comment by euu2021** [comment]
The [Jumper Addon](https://github.com/lilive/Freeplane-Jumper) gives you this option.
